backend/src/main/java/co/inventorsoft/scripty/controller/ProjectController.java

```diff
@@ -0,0 +1,52 @@
+package co.inventorsoft.scripty.controller;
+
+import javax.validation.Valid;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import co.inventorsoft.scripty.model.dto.ProjectDto;
+import co.inventorsoft.scripty.model.dto.StringResponse;
+import co.inventorsoft.scripty.service.ProjectService;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.experimental.FieldDefaults;
+
+/**
+ * @author lzabidovsky 
+ */
+@RestController
+@RequestMapping(value = "/users/{username}/projects")
```

Comment:
Replace `{username}` with `current` or `me` to get rid of additional authentication checks